DirectProgramming/DPC++FPGA/Tutorials/Features/device_global/sample.json

```
@@ -0,0 +1,61 @@
+{
+  "guid": "CDC9540F-B89F-499E-BC27-E8EC9EA3B2D7",
+  "name": "Private Copies",
+  "categories": ["Toolkit/oneAPI Direct Programming/DPC++ FPGA/Tutorials/Features"],
```

As you'll move this tutorial to the experimental folder:
```suggestion
  "categories": ["Toolkit/oneAPI Direct Programming/DPC++ FPGA/Tutorials/Features/experimental"],
```